deployments/deployments.json

```diff
@@ -31,7 +31,7 @@
     "AlephZero_Adapter": { "address": "0x6F4083304C2cA99B077ACE06a5DcF670615915Af", "blockNumber": 21131132 },
     "Ink_Adapter": { "address": "0x7e90a40c7519b041a7df6498fbf5662e8cfc61d2", "blockNumber": 21438590 },
     "Cher_Adapter": { "address": "0x0c9d064523177dBB55CFE52b9D0c485FBFc35FD2", "blockNumber": 21597341 },
-    "Lens_Adapter": { "address": "0x63AC22131eD457aeCbD63e6c4C7eeC7BBC74fF1F", "blockNumber": 22167069 },
+    "Lens_Adapter": { "address": "0x9a177ab7a1021def838f018bd09dbad7addb728e", "blockNumber": 22367550 },
```

Comment:
Adapter needs to be redployed with an updated ERC20 approval for the custom gas token.